doc/whats_new/v1.0.rst

```diff
@@ -70,6 +70,10 @@ Changelog
   in multicore settings. :pr:`19052` by
   :user:`Yusuke Nagasaka <YusukeNagasaka>`.
 
+- |Fix| Fixes incorrect multiple data-conversion warnings when clustering
+  boolean data. :pr:`19046` by :user:`Surya Prakash <jdsurya>` and
+  :user:`Nicolas Hug <NicolasHug>`.
```

Comment:
```suggestion
  boolean data. :pr:`19046` by :user:`Surya Prakash <jdsurya>`.
```

No need to credit me @jdsurya you did the work (but thanks 🙏 )

Reply:
cool :D